docker/Dockerfile

```diff
@@ -1,135 +1,95 @@
-FROM debian:9.8
+FROM debian:11
 MAINTAINER Pangeanic <info@pangeanic.com>
 ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get update && apt-get install -y --no-install-recommends apt-utils
 RUN apt-get update && apt-get install -y git
 
-# Install dependencies
-RUN apt-get install -y python3-pip && pip3 install virtualenv
-
-# Activate VirtualEnv in 'venv' directory
-# TODO (not working now): RUN virtualenv $ELASTICTM/venv && . $ELASTICTM/venv/bin/activate
-
 # Install packages
-RUN apt-get install -y libxml2-dev libxslt1-dev python-dev python3-bs4 libcurl4-openssl-dev apt-transport-https zip
+RUN apt-get install -y libxml2-dev libxslt1-dev python3-dev python3-bs4 libcurl4-openssl-dev apt-transport-https zip libtercpp-dev
 RUN apt-get install -y libblas-dev liblapack-dev libatlas-base-dev gfortran libssl-dev
-RUN apt-get install -y python-numpy python-scipy python-matplotlib ipython python-pandas python-sympy python-nose
-RUN apt-get update && apt-get install -y --no-install-recommends apt-utils software-properties-common
```

Comment:
**[blocker]** @higherstar  Me and Markus found that we should pin down a specific Python version. So we would like to ask you to specify the Python version when installing Python packages here. 

Currently it seems that the most recent `3.x` Python version is being installed. Moving forward we need the MAJOR and MINOR versions to be exact (meaning the semver structure of `MAJOR.MINOR.PATCH`).

Reply:
@DanBondarenko you mean, instead of python3, we need to specify exact version like python3.9?

So for example `python3.9-dev`, `python3.9-bs4`, ... ?

Reply:
@higherstar Yes, exactly :)

Reply:
@higherstar Seems that the command syntax for specifying the version might be different: https://askubuntu.com/questions/428772/how-to-install-specific-version-of-some-package

```
sudo apt-get install <package name>=<version>
```